pkg/eventprocessor/impl/eventprocessor_test.go

```diff
@@ -374,7 +377,7 @@ func setup(t *testing.T) (
 		queryRes := res.(*sqlstore.UserRows)
 		ret := make([]int64, len(queryRes.Rows))
 		for i := range queryRes.Rows {
-			ret[i] = (*queryRes.Rows[i][0].(*interface{})).(int64)
+			ret[i] = (*queryRes.Rows[i][0].(*sqlstore.UserValue)).Value().(int64)
```

Comment:
I think a similar change should be done [here](https://github.com/tablelandnetwork/go-tableland/blob/89637eb0cd23fe1fa1ca12a193ac37f94c3102d3/internal/router/controllers/user.go#L146).
Maybe can be double-checked with a test.

Reply:
Ah yea good catch. The current tests pass because the mocked data is returning `string`, not `UserData`. Hmm, I'll look into updating the mock data and that line or get to reading and considering your other comment about not using `UserData`.

Reply:
Fixed via strong typing of rows with `UserValue`